cmd/destroy.go

```diff
@@ -163,6 +165,9 @@ func newDestroyCmd() *cobra.Command {
 		"target", "t", []string{},
 		"Specify a single resource URN to destroy. All resources necessary to destroy this target will also be destroyed."+
 			" Multiple resources can be specified using: --target urn1 --target urn2")
+	cmd.PersistentFlags().BoolVar(
+		&forceTargets, "force-targets", false,
```

Comment:
Nit: can we call this `target-dependents` or something?

Reply:
i like that!